### PR TITLE
[JS] Added/Updated operators concept

### PR DESCRIPTION
**Pull request by ulyssear**
## What GitHub issue does this PR apply to?
   Part of issue #609 

## What changed and why?
   Added operators concept for ECMAScript 2009 and 2023, and updated this concept for ECMAScript 2020 and 2021.

## Checklist

- [x] I claimed any associated issue(s) and they are not someone else's
- [x] I have looked at the documentation to ensure I made any revisions correctly
- [x] I tested my changes locally to ensure they work
- [ ] (If editing Django) I have added or edited any appropriate unit tests for my changes

### CODE REVIEW

File: web/thesauruses/javascript/ECMAScript 2009/operators.json [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+        "not-implemented": true,
+        "name": "Percentage operator"
+      },
+      "equal_to": {
+        "name": "Equality operator",
+        "code": [
+          "a == b // loose equality",
+          "a === b // strict equality"
+        ],
+        "comment": "The `===` operator coerces the operands to the same type before comparing them. The `==` operator does not coerce the operands."
```
Comment (geekygirlsarah) [inline web/thesauruses/javascript/ECMAScript 2009/operators.json]: Isn't this backward? If `===` is strict equality, it _doesn't_ coerce the types.

File: web/thesauruses/javascript/ECMAScript 2020/operators.json [hunk 2]
```diff
@@ -1,204 +1,220 @@
...
-    "structure": "operators"
+    "structure": "operators",
+    "language_version": "ECMAScript 2020"
   },
   "concepts": {
     "addition": {
-      "code": "operand1 + operand2",
-      "name": "Addition operator"
+      "name": "Addition operator",
+      "code": "a + b"
```
Comment (geekygirlsarah) [inline web/thesauruses/javascript/ECMAScript 2020/operators.json:11]: Why are you redoing these? They were correct before.

**Review by geekygirlsarah — CHANGES_REQUESTED** [review]
I have a couple of comments and questions. Fix those (or tell me why they're right) and I can merge them in.

File: web/thesauruses/javascript/ECMAScript 2020/operators.json (hunk 2 above)
Comment (ulyssear) [reply web/thesauruses/javascript/ECMAScript 2020/operators.json:11]: I just shortened the expressions to make them more readable, but I can still replace them with names.

File: web/thesauruses/javascript/ECMAScript 2009/operators.json (hunk 1 above)
Comment (ulyssear) [reply web/thesauruses/javascript/ECMAScript 2009/operators.json]: Oh. sorry about that, I'll change the comment by the description on MDN about strict equality:
> The `===` operator checks whether its two operands are equal, returning a Boolean. Unlike the `==` operator, the strict equality operator always considers operands of distinct types to be different.

References :
- MDN page about strict equality: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality

File: web/thesauruses/javascript/ECMAScript 2020/operators.json (hunk 2 above)
Comment (geekygirlsarah) [reply web/thesauruses/javascript/ECMAScript 2020/operators.json:11]: That's fine. I'm not worried about short, but I guess if you're filling in others, it keeps it consistent.

**Review by geekygirlsarah — APPROVED** [review]
Alright, I think this is good. I'll merge it in!

By this point you should have earned your Hacktoberfest shirt! I hope you're registered!

**Pull request merged**